.github/workflows/pr.yml

```diff
@@ -35,6 +35,8 @@ jobs:
         run: yarn start:docker
       - name: yarn build
         run: yarn build
+      - name: yarn test
+        run: yarn test --exclude "**/e2e/**"
       - name: yarn start:contract
```

Comment:
Maybe better to stick this in the `unit` job? L23-L24

Reply:
oops! yes, that's where I meant to put it...

Reply:
This doesn't look right either: https://github.com/Agoric/dapp-offer-up/blob/70540a2de5c7dfa6e5b899adc8e21e64830286f6/.github/workflows/pr.yml#L12-L16, and Agoric/agoric-sdk#8636 is now closed.  Maybe just `18.x`, or `['18.x', '20.x']` since it's a quick job?

Reply:
I'll try it, but see also https://github.com/Agoric/documentation/pull/1065